andela/AndelaEatsUI
# #161079585 Implement add Engagement (UI)

#### What does this PR do?
Implement `Add` Engagement UI

#### Description of Task to be completed?
Ensure there is a UI for Admin to Add Engagements

#### How should this be manually tested?
- Log in to the app
- Switch to the admin dashboard from the user profile drop-down menu or [here](http://andelaeats-dev.andela.com:3000/admin/engagements)
- You should see a button with `Add Engagements`
- Click the button

#### What are the relevant pivotal tracker stories?
- [161079585](https://www.pivotaltracker.com/n/projects/2178749/stories/161079585)

#### Background Context
Nil

#### Screenshots (If Applicable)
<img width="1263" alt="screen shot 2018-11-14 at 4 53 40 pm" src="https://user-images.githubusercontent.com/21138774/48494300-e3557100-e82d-11e8-9f8d-7afb16577f06.png">
